Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Header.css';
+import HeaderTitle from './HeaderTitle';
+import HeaderContent from './HeaderContent';
```

Comment:
The organization looks nice. You're not organizing everything into the App. Good Job.